app/controllers/api/v1/messages_controller.rb

```diff
@@ -1,6 +1,14 @@
 module Api
   module V1
     class MessagesController < ApiController
+      def index
+        # TODO: Add Pagination and order filter
+        return render status: 404, nothing: true unless chat_room
+
+        messages = Message.where(chat_room: chat_room).order(created_at: :asc)
```

Comment:
For now it doesn't matter, but if we have paging, we need to retrieve newest messages first, and that means desc will be correct. In the UI results will be reversed before displaying. Do you agree?